content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/mpcie-4g-modem/content.md

```diff
@@ -312,16 +314,74 @@ fw_setenv is_on_carrier yes
 fw_setenv carrier_name max
 ```
 
-The previous commands are used to set environment variables, which we will use to set and use the needed overlays to link the mini PCIe interface under its profile. The overlays are as follows and set using the following command:
+The previous commands are used to set environment variables, which we will use to set and apply the needed overlays to link the USB modem interface under its profile. The overlays are as follows and set using the following command:
 
 ```bash
-fw_setenv overlays 'ov_som_lbee5kl1dx ov_som_x8h7 ov_carrier_enuc_bq24195 ov_carrier_max_usbfs ov_carrier_max_sdc ov_carrier_max_cs42l52 ov_carrier_max_pcie_mini'
+fw_setenv overlays 'ov_som_lbee5kl1dx ov_som_x8h7 ov_carrier_enuc_bq24195 ov_carrier_max_usbfs ov_carrier_max_sdc ov_carrier_max_cs42l52'
 ```
 
+***The `ov_carrier_max_pcie_mini` overlay is not required for USB modems such as the GNSS Global (EG25) and EMEA (EC200A-EU) variants of the Pro 4G Module.***
+
 Once the overlays are set, please reboot the Portenta X8 to ensure the configuration has been applied correctly.
 
 ### Connecting & Testing Network Connectivity
 
+#### GNSS Global EG25 Module
+
+For the **GNSS Global (EG25) Module**, you can configure the modem using **nmcli**:
+
+```bash
+nmcli c add type gsm ifname cdc-wdm0 con-name wwan0 apn hologram connection.autoconnect yes
+```
+
+If your SIM card requires a PIN, update the command as follows:
+
+```bash
+nmcli c add type gsm ifname cdc-wdm0 con-name wwan0 apn <APN> gsm.pin <PIN>
+```
+
+For instance, if you are using Vodafone in Italy, you can replace the `<APN>` field with `mobile.vodafone.it` and include the PIN number as well:
+
+```bash
+nmcli c add type gsm ifname cdc-wdm0 con-name wwan0 apn mobile.vodafone.it gsm.pin <PIN>
+```
+
+#### EMEA EC200A-EU Module
+
+For the **EMEA (EC200A-EU) Module**, which requires **mmcli**, use the following command to connect to the network after patching **ModemManager**:
+
+```bash
+mmcli -m 0 --simple-connect='apn=iot.1nce.net,ip-type=ipv4v6'
+```
+
+Ensure the `udev` rule remaps the USB `eth0` interface to `ec200aeu`. The rule can be applied as follows:
+
+```bash
+sudo nano /etc/udev/rules.d/99-remap-ec200aeu.rules
+```
+
+Add the following line:
+
+```bash
+SUBSYSTEM=="net", ACTION=="add", KERNEL=="eth0", NAME="ec200aeu"
+```
+
+Then reload the rules:
+
+```bash
+sudo udevadm control --reload-rules
+```
+
+```bash
+sudo udevadm trigger
+```
+
+#### For QMI Based Modems
+
+For **QMI based modems**, modems that use *Qualcomm MSM Interface*, following steps can help configure the modem, which involves managing the **Raw IP mode** and using **qmicli** for network control.
```

Comment:
Explain who are QMI modems or provide qmicli command to check the compatibility. Clarify that qmicli is an alternative to ModemManager (which is the tool you're using with mmcli) so is better to stop & disable ModemManager service to work in this mode

Reply:
Additional note: ec200aeu is NOT QMI compatible, it requires raw AT commands over usb serial interface. See https://python.quectel.com/en/products/ec200a-eu.